src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -355,6 +356,13 @@ class AddItemsPage extends Component {
       !item.statusCode &&
       parseInt(item.quantityRequested, 10) > 0 &&
       item.product);
+    // Here we have adding sent items to state, because we want to avoid
+    // sending a request with the same items few times
+    // (this problem occurs only during adding new items, because
+    // items which are already saved have theirs own IDs)
+    this.setState({
+      sentToSave: [...this.state.sentToSave, ...lineItemsToBeAdded],
```

Comment:
Instead of doing this, can you simply mark new rows (or edited) as `PENDING` before triggering the save?